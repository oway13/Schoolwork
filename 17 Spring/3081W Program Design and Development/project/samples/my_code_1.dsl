/* Reads in two matrices and multiplies them together */

main () {

matrix m = matrix_read( "./samples/sample_matrix_1.data" );
matrix n = matrix_read( "./samples/sample_matrix_2.data" );

matrix x = m * n;

print ( x );

}
