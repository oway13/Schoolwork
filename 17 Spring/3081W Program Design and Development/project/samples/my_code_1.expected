2 3
37  11  26  
39  15  24  
